src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -346,22 +345,40 @@ private ImmutableArray<ISymbol> GetSymbolsForExpressionOrStatementContext()
             var filterOutOfScopeLocals = _filterOutOfScopeLocals;
             if (filterOutOfScopeLocals)
             {
-                var contextNode = _context.LeftToken.GetRequiredParent();
                 filterOutOfScopeLocals =
                     !contextNode.IsFoundUnder<LocalDeclarationStatementSyntax>(d => d.Declaration.Type) &&
                     !contextNode.IsFoundUnder<DeclarationExpressionSyntax>(d => d.Type);
             }
 
-            var symbols = !_context.IsNameOfContext && _context.LeftToken.GetRequiredParent().IsInStaticContext()
-                ? _context.SemanticModel.LookupStaticMembers(_context.LeftToken.SpanStart)
-                : _context.SemanticModel.LookupSymbols(_context.LeftToken.SpanStart);
+            ImmutableArray<ISymbol> symbols;
+            if (_context.IsNameOfContext)
```

Comment:
>if (_context.IsNameOfContext) [](http://example.com/codeflow?start=12&length=29)

meant to be in this PR?

Reply:
so this is a followup to another pr (waiting on you to review as well) :)

Reply:
ok.  rereview.